movies/movies/CollectionView/View/ShowMoviesViewController.swift

```diff
@@ -261,56 +256,36 @@ final class ShowMoviesViewController: UIViewController {
     @objc private func changeListMovies(sender: UIButton) {
         switch sender {
```

Comment:
```suggestion
        switch sender {
```
вынеси логику в ViewModel